src/test/java/com/sugarcrm/candybean/automation/webdriver/WebDriverElementSystemTest.java

```diff
@@ -317,21 +321,23 @@ public void checkBoxSelectTest() throws Exception {
 
 	@Test
 	public void sendStringTest() throws Exception {
+        String garbageString = "This string is garbage that needs to be cleared.";
 		String searchString = "sugarcrm";
-		
-		// clear and send scenario
-		iface.go("http://www.duckduckgo.com/");
-		iface.getWebDriverElement(Strategy.ID, "search_form_input_homepage").sendString(searchString);
-		iface.getWebDriverElement(Strategy.ID, "search_button_homepage").click();
-		Assert.assertTrue(iface.getWebDriverElement(Strategy.PLINK, "SugarCRM").isDisplayed());
+        iface.go(elementsPage);
+
+        WebDriverElement textField = iface.getWebDriverElement(Strategy.ID, "textfield");
+
+		// clear and set scenario
+        textField.sendString(garbageString);
+        textField.sendString(searchString + Keys.TAB);
```

Comment:
Do we need to send the tab key here? I looks like all it does it focus the dropdown. Are we testing for something besides the strings matching?